perllib/FixMyStreet/Cobrand/Oxfordshire.pm

```diff
@@ -284,4 +289,98 @@ sub dashboard_export_problems_add_columns {
     });
 }
 
+sub defect_wfs_query {
+    my ($self, $params) = @_;
+
+    my $uri = URI->new("https://tilma.mysociety.org/proxy/occ/nsg/");
+    $uri->query_form(
+        %$params,
+        REQUEST => "GetFeature",
+        SERVICE => "WFS",
+        SRSNAME => "urn:ogc:def:crs:EPSG::4326",
+        TYPENAME => "WFS_DEFECTS_FOR_QUERYING",
+        VERSION => "1.1.0",
+        propertyName => 'ITEM_CATEGORY_NAME,ITEM_TYPE_NAME,REQUIRED_COMPLETION_DATE,ACTUAL_COMPLETION_DATE,APPROVAL_STATUS_NAME,SHAPE_GEOMETRY',
```

Comment:
`ACTUAL_COMPLETION_DATE` isn't used anywhere and can presumably be removed.

Reply:
Fixed in 75ca14a8cb5b11f2a48e3bc501159de4f83337b2